### PR TITLE
Feature/search cell rating view

### PR DESCRIPTION
**Pull request by alvare52**
# Description

Fixes UI stuff

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)

## Change Status

- [x] Complete, but not tested (may need new tests)
- [x] Incomplete/work-in-progress, PR is for discussion/feedback

# Checklist

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas

### CODE REVIEW

**Review by ghost — COMMENTED** [review]
You should remove unnecessary statements from Pull Request Template.

File: BetterReads/BetterReads/Views/SearchResultView.swift [hunk 1]
```diff
@@ -33,27 +51,56 @@ class SearchResultView: UIView {
...
+            } else {
+                star.image = UIImage(systemName: "star")
+                star.tintColor = .green
+            }
+        }
+    }
+    
     private func setUpSubviews() {
+        // DELETE
+        value = 3.0
```
Comment (ladybeitel) [inline BetterReads/BetterReads/Views/SearchResultView.swift]: Why does this say Delete?
Does this value need to be deleted?

File: BetterReads/BetterReads/Views/SearchResultView.swift [hunk 2]
```diff
@@ -33,27 +51,56 @@ class SearchResultView: UIView {
...
+        //imageView.widthAnchor.constraint(equalTo: imageView.heightAnchor, multiplier: 3 / 4).isActive = true
         
+        // mult for height used to be 1.5 of imageView.widthAnchor // widthAnchor used to be 0.25
+        // FIXME: (later) change imageView scale/size based on what image is passed in? cell size?
         imageView.contentMode = .scaleAspectFit
         imageView.backgroundColor = UIColor(red: 200.0/255.0,
                                             green: 200.0/255.0,
                                             blue: 200.0/255.0,
                                             alpha: 1.0)
+        imageView.tintColor = UIColor(red: 11.0/255.0, green: 28.0/255.0, blue: 124.0/255.0, alpha: 1.0)
```
Comment (ladybeitel) [inline BetterReads/BetterReads/Views/SearchResultView.swift:105]: What color is this?

**Review by ladybeitel — CHANGES_REQUESTED** [review]
Please review my comments.

File: BetterReads/BetterReads/Views/SearchResultView.swift (hunk 2 above)
Comment (alvare52) [reply BetterReads/BetterReads/Views/SearchResultView.swift:105]: Catalina Wine Mixer

File: BetterReads/BetterReads/Views/SearchResultView.swift (hunk 1 above)
Comment (alvare52) [reply BetterReads/BetterReads/Views/SearchResultView.swift]: temporary value, changed

**Comment by alvare52** [comment]
Reviewed comments, changes made

**Comment by ladybeitel** [comment]
[Search](https://trello.com/c/maOn01aG/65-search)

**Comment by ladybeitel** [comment]
[Search](https://trello.com/c/maOn01aG/65-search)

**Pull request merged**